doc/releases/release-notes-2.7.rst

```diff
@@ -50,6 +50,34 @@ Deprecated in this release
   :c:func:`bt_le_filter_accept_list_remove`
   :c:func:`bt_le_filter_accept_list_clear`
 
+Modified in this release
+
+* The following Bluetooth macros and structures in :file:`hci.h` have been
+  modified to align with the inclusive naming in the v5.3 specification:
+
+  * ``BT_LE_FEAT_BIT_SLAVE_FEAT_REQ`` is now :c:macro:`BT_LE_FEAT_BIT_PER_INIT_FEAT_XCHG`
```

Comment:
I am a bit concerned about the usage of `_PER_` for peripheral, as `_PER_` is also the abbreviation with for periodic in the context of periodic advertising. 

I don't have a good alternative though.

Reply:
The alternative is to use the full word "PERIPHERAL". I did think about that and was unsure, let me send an update with the extra changes that @thoh-ot listed and then we can see the new file contents as a whole.

Reply:
Ultimately I think in `hci.h` this is fine, because commands and events require looking at the spec every time anyway, so the confusion is very unlikely to happen.